### ISSUE TITLE
Strike-through needs a keyboard shortcut

### ISSUE THREAD

**Issue by Redsandro** [Zettlr/Zettlr#4571]
### Description

I use `~~strikethrough~~ a lot, and would like a [keyboard shortcut](https://www.zettlr.com/shortcuts). 

### Proposed Changes

We have `ctrl`+`i` and `ctrl`+`b`, but nothing for Strikethough.

### Caveats

`Ctrl`+`s` is already in use for saving a file.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Stable (but an older version)

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Operating System Version

Linux Mint 21

**Comment by Menn1s** [comment]
I agree on this! Especially since the markdown interpreter starting working more correctly and only registers strikethrough for double tildes before and after some text (it used to work just at the beginning of a bullet, albeit improperly).

I also see a good number of options in the contextual toolbar that pops up on highlighting! Wouldn't mind it there either.

<img width="305" alt="image" src="https://github.com/Zettlr/Zettlr/assets/38476987/61d78b11-4ac0-4ed9-b06f-c4670f84b788">

**Comment by ElDuderino321** [comment]
I'd like to see this implemented, as well!

**Comment by mw-852** [comment]
Yes me too. A button in the UI like bold and italic would also be appreciated
